file_paths.md

```diff
@@ -163,160 +163,195 @@ The package will create the following services `puppet`, `mcollective`,
 and `pxp-agent`, all running as `LocalSystem` by default. It will not
 create a `puppet` user or group.
 
-    Path                                      Setting                        3.x
-    C:\ProgramData                                                           n/a
+    Path                                          Setting                        3.x
+    C:\ProgramData\PuppetLabs *                                                  n/a
```

Comment:
It would be great to figure out how to use the original install path with spaces, otherwise there will be downstream docs changes, etc.